### ISSUE TITLE
Chain selection improvement

### ISSUE THREAD

**Issue by Tuxman** [Topl/Bifrost#520]

**Comment by scasplte2** [comment]
Problems appear to be fixed with chain selection and integration of previous updates. Commit https://github.com/Topl/Bifrost/pull/569/commits/337420e57f684f601d1aaf55f31841904037c8c3 should be ready to merge into dev. We will wait on Nick fixing the issues he has encountered with Netty to push to master.
